config/default/manager_auth_proxy_patch.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1
```

Comment:
this URL will go away soon I think. Fine to do this now as it matches what we do in next gen. But soon we should revisit these and either use the downstream image for kube-rbac-proxy or perhaps look at options to remove it

Reply:
to be clear, no action needed in this PR at this time